src/components/Form/Form.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from 'react';
+import usersFromServer from '../../api/users';
+import { EnrichedTodo } from '../../types/EnrichedTodo';
+import { getUserById } from '../../services/user';
+
+type Props = {
+  onSubmit: (todo: EnrichedTodo) => void;
+};
+
+export const Form: React.FC<Props> = ({ onSubmit }) => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    if (title === '' || userId === 0) {
+      return;
+    }
+
+    onSubmit({
+      id: 0,
+      title,
+      completed: false,
+      userId,
+      user: getUserById(userId),
+    });
+
+    setTitle('');
+    setUserId(0);
+  };
+
+  return (
+    <form action="/api/todos" method="POST">
```

Comment:
The `action` and `method` attributes are not necessary here as the form submission is handled by React. You can remove these attributes to avoid confusion.